ACMILabs/works-embeddings
# #7 Fix touchscreen UX bugs

Resolves #7 

There are some touchscreen/calibration issues, so let's resolve them.

### Acceptance Criteria
- [x] Home page scrollbars on wiggle
- [x] Scrollbars on popup
- [x] Popup - any tap closes the popup
- [x] Large circle a bit smaller
- [x] Titles limit to 3 lines
- [x] Images fade in
- [x] Add a `/api/taps/` API for the lens reader to send taps to
- [x] Add a `/api/tap-source/` API for the frontend to subscribe to tap responses

### Relevant design files
* None

### Testing instructions
1. Test the touch updates on a production device
1. To test the Taps API with a lens reader, add `LENS_READER_TAPS_API` and `AUTH_TOKEN` to your `config.env`
1. Set your Lens Reader environment variables: `TARGET_API_ENDPOINT=http://<LAPTOP_IP>:8081/api/` and `XOS_TAPS_ENDPOINT=http://<LAPTOP_IP>:8000/api/taps/`
1. Start XOS: `make up`
1. Start this server: `make up`
1. Visit: http://localhost:8081 and then tap on your lens reader
1. Visit the Lens debug page to see the taps: http://localhost:8000/lens/
